From 29f9c792476dd01ef0dd32c0d7222d2da31ea0da Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 21 Nov 2016 20:55:37 +0800
Subject: [PATCH 1283/5242] MLK-13487-1 ARM: dts: imx7ulp: add PTC1 pin as
 GPIO

commit  07151a50680e085cb8e95974b1e488bf652a2179 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add PTC1 pin as GPIO on i.MX7ULP SOM board, it is
to control NVCC_DRAM_SW during suspend/resume.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index badbb29..74ca200 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -33,6 +33,7 @@
 		pinctrl_hog_1: hoggrp-1 {
 			fsl,pins = <
 				IMX7ULP_PAD_PTC10__PTC10	0x30100
+				IMX7ULP_PAD_PTC1__PTC1		0x20100
 			>;
 		};
 
-- 
1.7.9.5

